external/curl/Make.defs

```diff
@@ -0,0 +1,56 @@
+###########################################################################
+#
+# Copyright 2016 Samsung Electronics All Rights Reserved.
```

Comment:
Year should be 2018. Please check all of added files by Samsung.

Reply:
Okey